app/models/comment.rb

```diff
@@ -0,0 +1,19 @@
+class Comment < ApplicationRecord
+  belongs_to :ticket
+  has_many :children, class_name: 'Comment', foreign_key: 'parent_id'
+  belongs_to :parent, class_name: 'Comment', optional: true
+  validates :worker_id, :ticket_id, presence: true
+  validates :message,  length: {minimum: 3}
+
+  def check_time(hours)
```

Comment:
method_naming not self explanatory
`check_time` what does it check?
I also having torubles with method namings, at least you should put comments explaining what this method does